cluster-provision/k8s/1.18/nodes.sh

```diff
@@ -25,11 +25,11 @@ done
 if [ -f /etc/sysconfig/kubelet ]; then
     # TODO use config file! this is deprecated
```

Comment:
Is this comment still relevant?

Reply:
yes, again refactor of kubelet will be done on a follow PR
it shouldnt block this one, we have plenty to do, and people are waiting for this PR which doesnt break anything,
i did some refactor under the scope of this + calico + sonobuoy and i think we shouldnt block anymore please